OpenSim/Simulation/Model/Ligament.h

```diff
@@ -162,6 +188,10 @@ OpenSim_DECLARE_CONCRETE_OBJECT(Ligament, Force);
         return values;
     }
 
+    OpenSim_DECLARE_PROPERTY(path, AbstractPath,
```

Comment:
Should this be separate from the other `DECLARE_PROPERTY`s?

Reply:
I moved this back with the other `DECLARE_PROPERTY`s. (I think I did this out of habit based on how we use properties in Moco, but it's not really necessary here).